src/openforms/contrib/referentielijsten/api/serializers.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import serializers
+
+from openforms.api.utils import mark_experimental
+
+
+@mark_experimental
```

Comment:
I'm not sure what exactly about this is so experimental that it needs to be marked accordingly. They're private API endpoints, so we can tweak without breaking backwards compatibility guarantees.